content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,78 @@
+---
+authors: ["veronica-bolognesi"]
+comments: true
+date: "2020-09-10"
+draft: false
+share: true
+categories: [English, Agile, Scrum, Kanban]
+title: "PHPDay 2020"
+
+languageCode: "en-EN"
+type: "post"
+---
+On sept 8th the [2020 edition of PHPDay](https://2020.phpday.it/) was held in a remote fashion.
+
+Having experienced the 2019 edition (indeed my first one, despite being a PHP developer from eons!) I should say that it was a little difficult to maintain the concentration for such a long event, but the effort was well worth of, given the level of both lecturers and talks.
+
+I would like to thanks the folks at [Grusp](https://www.grusp.org/) for their passionate work. Try to sustain them!
+
+What follows is a mere recap of the event, with my little bit of comments. 
+
+
+# Things I Wish I Knew When I Started Playing Developer
+## by Gary Hockin ([@GeeH](https://twitter.com/GeeH))
+
+
+A trip into developers' mindset from a ludic (and hence, engaging) viewpoint. Gary is a natural born speaker ;) and this talk deserves to be seen by anyone interested in programming, no matter their background and interests, at least once. 
+
+#What's new in PHP 8 
+## Derick Rethans ([@derickr](https://twitter.com/derickr))
+
+Personally, I did not like the style of the presentation (not the theme of slides, but how they got presented! :D) but the information provided is very important and so you should take a look here: https://derickrethans.nl/talks/php-phpday20 
+
+#Symfony 5, the new bit 
+## Andreas Hucks ([@meandmymonkey](https://twitter.com/meandmymonkey))
+
+A deep dive into Symfony 5 features and enhancements. Knowing where the framework is heading will make us take better-informed decisions in our day to day work.
+
+#Leveraging Typed Exceptions for Cleaner Error Handling 
+## Chris Holland ([@chrisholland](https://twitter.com/chrisholland))
+
+A talk suited mostly to entry-level/mid-level developers, but indeed useful to learn (or recap) how to establish a lifelong relationship with exceptions: they are not enemies, they can help us contextualizing the scope in which they occur and so speed up the debugging process.
```

Comment:
What about it?

This speech is mainly intended for entry-level or mid-level developers. However, it is very useful to learn (or recap) how to build a long-lasting relationship with exceptions. In fact, exceptions are not our enemies, they can help us contextualize the scope in which they occur. Therefore, exceptions can speed up the debugging process.

Reply:
Ok, but I'll change from present to past tense: this speech was